### PR TITLE
baremetal disable lvm mappings wipe additional disks

### PR DESCRIPTION
**Pull request by ipochi**
This PR disables LVM partitions so that any detected LVM partitions are not automatically 
activated on bootup disrupting the installation and wipes any additional disks attached to the
machine before proceeeding 

This PR is in a series of multiple PRs that will be created to ease the review/acceptance/merge of a large PR #1402

### CODE REVIEW

File: assets/terraform-modules/matchbox-flatcar/templates/install.yaml.tmpl [hunk 1]
```diff
@@ -30,6 +30,22 @@ storage:
       contents:
         inline: |
           #!/bin/bash -ex
+          # unmount any leftovers at the beginning before we modify the disk the mount uses
+          umount /tmp/oemfs || true
```
Comment (pothos) [inline assets/terraform-modules/matchbox-flatcar/templates/install.yaml.tmpl:34]:
```suggestion
```
I will put these it the PR that introduces the /tmp/oemfs mount

Comment (pothos) [reply assets/terraform-modules/matchbox-flatcar/templates/install.yaml.tmpl:34]: It's now part of https://github.com/kinvolk/lokomotive/pull/1489/files

**Review by pothos — DISMISSED** [review]

**Review by pothos — APPROVED** [review]

**Review by pothos — APPROVED** [review]

**Comment by invidian** [comment]
I'm not sure if it makes sense for me to review this PR. Perhaps someone more involved in maintenance should do this.

**Review by surajssd — APPROVED** [review]
LGTM :tada:

**Pull request merged**